root/static/scripts/common/MB.js

```diff
@@ -35,9 +35,6 @@ const MB: {[string]: *} = {
 
     // Hold constants for knockout templates that depend on globals.
     constants,
-
-    // Deprecated reference needed by knockout templates
-    i18n: require('./i18n')
```

Comment:
The problem with globals like this is that technologies like Webpack can only look at the imports to create a dependency tree and decide how to bundle things. It would have no idea where this global variable is used, thus we have to include i18n.js on every page (which maybe is necessary in this example, but isn't in others - it can't tell either way).